packages/perseus/src/widgets/__tests__/__snapshots__/interactive-graph.test.ts.snap

```diff
@@ -3250,6 +3250,24 @@ exports[`snapshots should render correctly with locked lines 1`] = `
             <div
               class="default_xu2jcg-o_O-inlineStyles_16155wj"
             >
+              <span
+                style="position: absolute; left: 400px; top: 200px; font-size: 14px; transform: translate(7px, -50%);"
+              >
+                <span
+                  class="mock-TeX"
+                >
+                  x
+                </span>
+              </span>
+              <span
+                style="position: absolute; left: 200px; top: -24px; font-size: 14px; transform: translate(-50%, -7px);"
+              >
+                <span
+                  class="mock-TeX"
```

Comment:
Is it a problem that the class is "mock-TeX" here?

Reply:
No, that's fine. The TeX component being used here is a lightweight version just for testing. Webapp will inject its own TeX component.